example-deployment/docker/minimal.dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ros:foxy-ros-base-focal
+
+RUN apt-get update && apt-get install -y curl && \
+  curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  apt-get update && apt-get install -y nodejs python3-pip
+
+RUN pip3 install pipenv
+
+COPY . /root/rmf-web
+SHELL ["bash", "-c"]
+
+RUN npm config set unsafe-perm && cd /root/rmf-web && \
+  CI=1 npm install -g lerna@4 && lerna bootstrap --scope=minimal
+
+RUN cd /root/rmf-web/packages/minimal && \
+  PUBLIC_URL='/minimal' \
+  REACT_APP_REPORTING_SERVER='https://example.com/logserver/api/v1' \
+  REACT_APP_AUTH_PROVIDER='keycloak' \
+  REACT_APP_KEYCLOAK_CONFIG='{ "realm": "rmf-web", "clientId": "minimal", "url": "https://example.com/auth" }' \
+  npm run build
+
+###
+
+FROM nginx:stable
+COPY --from=0 /root/rmf-web/packages/minimal/build /usr/share/nginx/html/minimal
```

Comment:
will probably need to patch this https://github.com/open-rmf/rmf-web/pull/460 in, so that you do not encounter infinite redirects.